openedx_caliper_tracking/tests/test_caliper_delivery.py

```diff
@@ -25,11 +25,14 @@ def setUp(self):
         )
         with open(input_file) as current:
             self.event = json.loads(current.read())
+        patcher = mock.patch('openedx_caliper_tracking.processor.deliver_caliper_event')
+        self.delivery_mock = patcher.start()
+        self.addCleanup(patcher.stop)
 
-    @mock.patch(
-        'openedx_caliper_tracking.processor.deliver_caliper_event'
+    @override_settings(
+        FEATURES={}
```

Comment:
Just to get rid of unnecessary checks.

Reply:
Why not simply set the corresponding flag to false?

Reply:
Why not simply set the corresponding flag to False. That would be more readable. What do you think?

Reply:
The next test is doing the same thing that you mentioned.

Reply:
One test is without settings given and the other one is with the flag unset.